app/utils/error_handlers.py

```diff
@@ -0,0 +1,98 @@
+import typing as tp
+
+from flask import Response, jsonify
+from pydantic.error_wrappers import ValidationError
+from sqlalchemy.exc import IntegrityError
+
+from utils.exceptions import (
+    LimitOffsetError,
+    SortException,
```

Comment:
where is this raised?

Reply:
in next pr

Reply:
then you should include it in the future PR and not now